.github/CODEOWNERS

```
@@ -215,6 +215,7 @@ Doc/c-api/stable.rst          @encukou
 /Doc/library/idle.rst         @terryjreedy
 
 **/*typing*                   @JelleZijlstra @AlexWaygood
+**/*annotationlib*            @JelleZijlstra
```

We don't have much sorting going on in this file, but let's apply a bit of arbitrary order:

```suggestion
**/*annotationlib*            @JelleZijlstra
**/*typing*                   @JelleZijlstra @AlexWaygood
```